### PR TITLE
add some packages for xfce4-goodies

### PR DESCRIPTION
**Pull request by Yisus7u7**
do not merge the changes yet, I will add more packages every time I have time and when I finish I will let you know.

### CODE REVIEW

**Comment by Yisus7u7** [comment]
@xeffyr 
the error is not mine, it is from the compiler.
```bash
termux - building xfce4-goodies for arch aarch64...
/home/builder/termux-packages/scripts/build/termux_step_create_debian_package.sh: line 49: AR: unbound variable
Error: Process completed with exit code 1.
```

**Comment by Yisus7u7** [comment]
Screenshot_20211122-130542

**Comment by Yisus7u7** [comment]
@xeffyr You can now merge the changes.

Actual state:
- Added a large part of xfce4-goodies
- Unable to create xfce4-goodies meta package due to a bug in termux_step_create_debian_package.sh
- file-roller fails during "ninja install"
- it is recommended to disable the xarchiver package as it is useless, it does not work.
- xfce4-screenshot has a rare error with help2man, this does not happen on the device.

*Disabled packages*
- file-roller
- xfce4-goodies
- xfce4-screenshoter
- libdbusmenu-gtk3

**Comment by Yisus7u7** [comment]
I'm very exhausted, I hope this helps

File: disabled-packages/libdbusmenu-gtk3/build.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+TERMUX_PKG_SHA256=41298b926573419f21864205317461750b833c596af6ab0bd206e13336f8cee3
+TERMUX_PKG_DEPENDS="gtk3, glib, atk, libcairo, gdk-pixbuf, harfbuzz, pango, xsltproc, json-glib"
+TERMUX_PKG_BUILD_DEPENDS="gtk-doc"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--enable-introspection=no --disable-vala"
+# autoreconf fails by gtk-doc
+# in a build on device this is fixed by 
+# installing gtk-doc
+termux_step_pre_configure(){
+	export PATH=$PATH:$TERMUX_PREFIX/bin
```
Comment (xeffyr) [inline disabled-packages/libdbusmenu-gtk3/build.sh:16]: Don't use `$TERMUX_PREFIX/bin`. You can't use cross-compiled tools in builder image.

File: disabled-packages/xfce4-goodies/build.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+TERMUX_PKG_HOMEPAGE=https://goodies.xfce.org/
+TERMUX_PKG_DESCRIPTION="this meta-package will install various utilities and programs for the XFCE desktop environment (incomplete!)"
+TERMUX_PKG_LICENSE="Public Domain"
+TERMUX_PKG_MAINTAINER="Yisus7u7 <jesuspixel5@gmail.com>"
+TERMUX_PKG_VERSION=4.14.0
+TERMUX_PKG_SKIP_SRC_EXTRACT=true
+TERMUX_PKG_DEPENDS="xfce4-terminal, xfce4-appfinder, xfce4-calculator-plugin, xfce4-clipman-plugin, xfce4-datetime-plugin, xfce4-dict, xfce4-eyes-plugin, xfce4-mailwatch-plugin, xfce4-netload-plugin, xfce4-notes-plugin, xfce4-notifyd, xfce4-panel-profiles, xfce4-taskmanager, xfce4-timer-plugin, xfce4-wavelan-plugin, xfce4-whiskermenu-plugin, thunar-archive-plugin, parole, xarchiver, leafpad"
+TERMUX_PKG_CONFLICTS="xfce-goodies"
+TERMUX_PKG_REPLACES="xfce-goodies"
+TERMUX_PKG_PROVIDES="xfce4-plugins"
```
Comment (xeffyr) [inline disabled-packages/xfce4-goodies/build.sh:12]: Package conflicts with itself, replaces and provides itself? Are you sure that these fields should be set for the newly added package?

**Review by xeffyr — CHANGES_REQUESTED** [review]

**Comment by Yisus7u7** [comment]
@xeffyr You can now merge the changes.

**Comment by Yisus7u7** [comment]
> when can we download this? thanks for the good work

just search for the name of the plugin you need and install it with pkg

**Comment by Yisus7u7** [comment]
Or run:

`pkg install xfce4-*`

**Pull request merged**